lib/beacon/live_admin/live/page_editor_live/variants.ex

```diff
@@ -0,0 +1,228 @@
+defmodule Beacon.LiveAdmin.PageEditorLive.Variants do
+  @moduledoc false
+  use Beacon.LiveAdmin.PageBuilder
+
+  import Beacon.LiveAdmin.PageEditorLive.Edit, only: [template_error: 1]
+
+  alias Beacon.LiveAdmin.Content
+
+  def menu_link("/pages", :variants), do: {:submenu, "Pages"}
+  def menu_link(_, _), do: :skip
+
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
```

Comment:
You can remove these lines if you want since the default implementation does return `{:ok, socket}` already: https://github.com/BeaconCMS/beacon_live_admin/blob/37fb459638fb683f84410e9d7b1397250e377772/lib/beacon/live_admin/page_live.ex#L48

```suggestion
```